stdlib/tkinter/ttk.pyi

```diff
@@ -139,8 +139,14 @@ class Style:
         self,
         elementname: str,
         etype: Literal["image"],
-        default_image: tkinter._ImageSpec,
-        /,
+        # This is technically positional-only and others above are positional-or-keyword.
+        # But as this isn't optional, the ones above must also be positional-only.
+        # However, this is inconsistent with the implementation (as it technically has the ones above as positional-or-keyword
+        # and this one in *args and then it checks that this is present but to get this into the *args, the previous 2 must
+        # also be given positionally so are in practise positional-only). Therefore, stubtest complains.
+        # (I'm not sure if this is the best way to handle this - ignoring the
+        # stubtest error would result in stubs that actually match the code)
+        _default_image: tkinter._ImageSpec,
         *imagespec: _ImageStatespec,
```

Comment:
Kinda awkward but you could do `*imagespec: *tuple[tkinter._ImageSpec, *tuple[_ImageStatespec, ...]]`. Not sure if this would be problematic to use though for code completion though.

Reply:
I think you should just ignore the stubtest error.